libs/@guardian/libs/project.json

```diff
@@ -34,7 +34,7 @@
 			"outputs": ["{workspaceRoot}/coverage/libs/@guardian/libs"],
 			"options": {
 				"jestConfig": "libs/@guardian/libs/jest.config.ts",
-				"passWithNoTests": true
+				"passWithNoTests": false
```

Comment:
Assume this is intentional as it's `true` for other projects? (Clearly there _are_ tests for libs anyway.)

Reply:
doh! that slipped through from #1131

Reply:
If #1131 is going in next then there's probably no point changing this back 👍